source/common/network/connection_impl.cc

```diff
@@ -644,6 +644,9 @@ bool ConnectionImpl::bothSidesHalfClosed() {
   return read_end_stream_ && write_end_stream_ && write_buffer_->length() == 0;
 }
 
+std::chrono::milliseconds ConnectionImpl::delayedCloseTimeout() const { 
```

Comment:
nits: I don't think a member function is necessary, can you just use delayed_close_timeout_ directly in where this method is currently called?

Reply:
Nice catch. Replaced already